### PR TITLE
Remove overzealous obsoletes from packages

### PR DESCRIPTION
**Pull request by pcreech**
python-aiodns through python-certifi

### CODE REVIEW

File: packages/python-ansible-builder/python-ansible-builder.spec
Comment (evgeni) [inline packages/python-ansible-builder/python-ansible-builder.spec]: this one contains things in `_bindir` and *needs* the obsoletes.

File: packages/python-bindep/python-bindep.spec
Comment (evgeni) [inline packages/python-bindep/python-bindep.spec]: bindir!

File: packages/python-ansible-builder/python-ansible-builder.spec
Comment (pcreech) [reply packages/python-ansible-builder/python-ansible-builder.spec]: good catch!  I thought I skipped them all, guess not.  Will fix when I come back around to clean up

File: packages/python-bindep/python-bindep.spec
Comment (pcreech) [reply packages/python-bindep/python-bindep.spec]: See previous comment

**Review by evgeni — CHANGES_REQUESTED** [review]
the packages that have things in `_bindir` and `_sbindir` should keep their obsoletes.

File: packages/python-bindep/python-bindep.spec
Comment (pcreech) [reply packages/python-bindep/python-bindep.spec]: er... "%exclude %{_bindir}/bindep"  - but... we're not including a bindir, we're excluding it?

Comment (pcreech) [reply packages/python-bindep/python-bindep.spec]: per irc convo, we're gonna keep the Obsoletes on anything with bindir for now, and can remove as we find need to.

File: packages/python-ansible-builder/python-ansible-builder.spec
Comment (pcreech) [reply packages/python-ansible-builder/python-ansible-builder.spec]: removed

File: packages/python-bindep/python-bindep.spec
Comment (pcreech) [reply packages/python-bindep/python-bindep.spec]: removed

**Review by Odilhao — APPROVED** [review]

**Comment by Odilhao** [comment]
I reviewed all files, we should be good to merge this after the scratch pipeline, before the merge I'll clean the build on copr to avoid obsolete problems

**Pull request merged**
